src/main/java/Main.java

```diff
@@ -0,0 +1,17 @@
+// overloading in methods
+class A {
```

Comment:
please write a separate class. 
And write unit test to test it.
Make it an habit.
Don't just copy and past.